spec/sql_enum_spec.rb

```diff
@@ -11,12 +9,12 @@
 
     before do
       SqlEnum.configure { |config| config.default_prefix = true }
-
       define_model('Task',
                    status: [:enum, limit: statuses, default: 'pending'],
                    priority: [:enum, limit: priorities]) do
         sql_enum :status
         sql_enum :priority, _prefix: false, _suffix: true
+        sql_enum :status # duplicate should be no-op
```

Comment:
Is there a way to directly test that this is in fact a no-op? I'm worried about the possibility of someone re-declaring using different options, then not understanding why their declaration doesn't work.

In fact, should this `raise`?

Reply:
Open to suggestions, this was from @dpep's PR, so maybe he wants to opine...